### PR TITLE
New Extension: Knowwa

### PR DESCRIPTION
**Pull request by d0utone**
## Description

This raycast extension enables fast access to document template created in Knowwa. 

Knowwa (knowwa.com) is a tool to improve the workflow when working on documents (Google Docs, Google Sheets etc) with multiple people. A user can add simple template tags (i.E. {first-name}) to a document and add this template to the platform by pasting the link to the google doc. Knowwa generates a  list of required inputs, tracks the progress, nudges users who did not provide input and handles access control.

With the Raycast extension, new documents can be directly created and viewed directly which is amazing!

## Screencast

[Youtube Video](https://www.youtube.com/watch?v=ZiBwCv0JKnU)

## Checklist

- [X] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [X] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [X] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [X] I checked that files in the `assets` folder are used by the extension itself
- [X] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/knowwa/src/modules/NewDoc/NewDocForm.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      toast.style = Toast.Style.Success;
+      toast.title = "Document Generated and URL copied to clipboard!";
+    } catch (err) {
+      toast.style = Toast.Style.Failure;
+      toast.title = "Could not generate Document";
+    }
+  };
+
+  return (
+    <Form
```
Comment (pernielsentikaer) [inline extensions/knowwa/src/modules/NewDoc/NewDocForm.tsx:53]: Are there some required fields? You might want to validate with [Validation](https://developers.raycast.com/api-reference/user-interface/form#validation)

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you 😊

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/knowwa/src/modules/NewDoc/NewDocForm.tsx (hunk 1 above)
Comment (d0utone) [reply extensions/knowwa/src/modules/NewDoc/NewDocForm.tsx:53]: The API of Knowwa currently does not support marking required fields, but I will keep it on the radar for next releases. The good news is, the submission will never fail due to missing or wrongly formatted fields yet.

**Comment by d0utone** [comment]
Hey @pernielsentikaer 🤙 

Thanks for the swift feedback and the correction! I committed the suggested change and will keep validation on the roadmap 🛣️.

**Review by pernielsentikaer — APPROVED** [review]
Hello 👋

I gave it a quick spin, works as expected, approved 🔥

**Pull request merged**